main.go

```diff
@@ -304,7 +309,16 @@ func backendMain() error {
 						return
 					case payments := <-client.ReceivedPayments():
 						for _, payment := range payments {
-							doDeposit(engine, payment, asset)
+							if isExchangePayment(payment) {
```

Comment:
It is okay for now, but in future we would need to have no difference between systems we are notifing